docs/layers.md

```diff
@@ -804,7 +804,7 @@ Places with `kind` values of `continent`, `country`, with others added starting
 * `name`
 * `id`: The `osm_id` from OpenStreetMap or Natural Earth id
 * `kind`: normalized values between OpenStreetMap and Natural Earth
-* `population`: population integer values from OpenStreetMap or Natural Earth's maximum population value.
+* `population`: population integer values from OpenStreetMap or Natural Earth's minimum population value(`pop_min`) if the place has a join from OpenStreetMap and NatualEarth or an estimate value hardcoded in the transform function `tags_set_ne_pop_min_max_default` in this repo.
```

Comment:
`or an estimate based on the type of place.`

Don't need to get that technical for general audience documentation.